### PR TITLE
Autopark lvl13

### PR DESCRIPTION
**Pull request by macevilamv**

### CODE REVIEW

File: src/main/java/by/incubator/autopark/infrastructure/config/impl/JavaConfig.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Override
+    public Scanner getScanner() {
+        return this.scanner;
+    }
+}
+
+
+
```
Comment (AntonNovosad) [inline src/main/java/by/incubator/autopark/infrastructure/config/impl/JavaConfig.java]: Many empty rows. Remove please

Comment (macevilamv) [reply src/main/java/by/incubator/autopark/infrastructure/config/impl/JavaConfig.java]: fixed

**Review by AntonNovosad — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -13,4 +13,22 @@
...
+    <dependencies>
+        <!-- https://mvnrepository.com/artifact/org.projectlombok/lombok -->
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.24</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.reflections/reflections -->
```
Comment (AlexeyKatomachin) [inline pom.xml]: please, remove this not needed commits

File: src/main/java/by/incubator/autopark/parsers/ParserVehicleFromFile.java [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        vehicleId = Integer.parseInt(rentParametersBuffer[CONFIG.get("RENT-CSV_CAR_ID")]);
+        rentDate = simpleDateFormat.format(simpleDateFormat.
+                parse(rentParametersBuffer[CONFIG.get("RENT-CSV_DATE")]));
+        cost = Double.parseDouble(rentParametersBuffer[CONFIG.get("RENT-CSV_COST")]);
+
+        return new Rent(vehicleId, rentDate, cost);
+    }
+
+    private Vehicle createVehicle(String csvStringOfVehicle) throws IOException, ParseException {
+        double mass;
```
Comment (AlexeyKatomachin) [inline src/main/java/by/incubator/autopark/parsers/ParserVehicleFromFile.java]: Optional:

maybe we can somehow beatify it 
for example int id = Integer.parseInt(...);

Comment (macevilamv) [reply src/main/java/by/incubator/autopark/parsers/ParserVehicleFromFile.java]: fixed

File: pom.xml (hunk 2 above)
Comment (macevilamv) [reply pom.xml]: removed

**Review by AlexeyKatomachin — APPROVED** [review]

**Pull request merged**